server/routes/initialAppointment/index.ts

```diff
@@ -32,6 +34,7 @@ export default function Index({ licenceService, conditionService }: Services): R
       routePrefix(path),
       roleCheckMiddleware(['ROLE_LICENCE_CA', 'ROLE_LICENCE_RO']),
       fetchLicence(licenceService),
+      licenceKindCheckMiddleware(LicenceKind.VARIATION),
```

Comment:
This was previously covered by `hardStopCheckMiddleware`, but has now been split into its own middleware.